frontend/src/Button.js

```diff
@@ -0,0 +1,15 @@
+import React, {Component} from 'react';
+import './index.css';
+
+class Button extends Component {
```

Comment:
Would prefer if this wasn't a hook.

Also, there seems to be no state used here, so I think this should probably be refactored into a functional component instead.